src/steps/fetch-bounties/converters.ts

```diff
@@ -60,16 +63,17 @@ export function createBountySubmissionEntity(data: BountySubmission) {
         name: data.title,
         displayName: data.title,
         category: 'bug-bounty',
-        description: data.description_markdown,
+        description: data.description_markdown || undefined,
         descriptionMarkdown: undefined,
         details: data.extra_info_markdown,
         extraInfoMarkdown: undefined,
         createdOn: parseTimePropertyValue(data.submitted_at),
-        references: data.vulnerability_references_markdown,
+        references: getReferences(data.vulnerability_references_markdown),
         vulnerabilityReferencesMarkdown: undefined,
-        numericSeverity: data.priority,
+        numericSeverity: data.priority || 5,
```

Comment:
Why are we doing this?

Reply:
It's weird looking, agree - basically `data.priority` can be `null` and we can't assign `null` or `undefined` to `numberSeverity` field so we "looked for some default value". What would you do here in this situation?